### DISCUSSION TITLE
push buttons

### DISCUSSION THREAD

**Discussion by maicolgenz** [salman0ansari/whatsapp-api-nodejs#709] [General]
When sending buttons it works well, but on many devices it does not appear, only on webwhatsapp.
I've already checked on a device, and it's on the latest version of whatsapp.
What could it be, something I did or is it the same api bug?

Tests with the meta api works normal for the same phone.

**Comment by salman0ansari** [comment]
Meta blocked buttons and list messages from Non-Official API.
read this for more info
https://github.com/WhiskeySockets/Baileys/issues/56#issuecomment-1542947921
